.gitignore

```
@@ -55,3 +56,5 @@ rsconnect/
 # Emacs
 *~
 \#*
+
+venv
```

Is this for Python virtual envs?

I don't currently keep mine within the project folder (rather a central location within my `$HOME`) so I'm not sure if this is a file or directory. If its a directory then I think the convention is to use...

```suggestion
venv/
```